core-common/src/main/kotlin/com/twitter/rules/core/util/Composables.kt

```diff
@@ -28,6 +49,15 @@ private val KtCallExpression.containsComposablesWithModifiers: Boolean
         .filter { it.isNamed() }
         .any { it.getArgumentName()?.text == "modifier" }
 
+/**
+ * This is a denylist with common composables that emit content in their own window. Feel free to add more elements
```

Comment:
probably don't need the feel free sentence in the comments.

Reply:
I copied the doc style from the other list. Personally I would not have used any docs here at all. 
Feel free to edit/ remove.

Reply:
true...matches other comment 😄